src/styles/companies.css

```diff
@@ -0,0 +1,91 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;1,300&display=swap");
+
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.companies-homepage {
+  background: #457b9d;
+  color: #fff;
+  padding: 5px;
+}
+
+.loading {
+  font-family: "Lato", sans-serif;
+  font-style: italic;
+}
+
+.search {
+  display: flex;
+  gap: 10px;
+  margin-bottom: 15px;
+}
+
+.search-company {
+  font-family: "Lato", sans-serif;
+}
+
+.input-icon {
+  display: flex;
+  align-items: center;
+}
+
+.search-input {
+  min-width: 250px;
+  padding: 5px;
+  border: 1px solid #333;
+  outline: none;
+  border-bottom-left-radius: 5px;
+  border-top-left-radius: 5px;
+}
+
+.search-icon {
+  font-size: 26px;
+  color: #333;
+  border: 1px solid #333;
+  border-top-right-radius: 5px;
+  border-bottom-right-radius: 5px;
+  background: #fff;
+}
+
+.companies-h1 {
+  font-family: "Lato", sans-serif;
+  font-size: 35px;
+  text-align: center;
+  margin-bottom: 10px;
+}
+
+.companies {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-auto-rows: auto;
+}
+
+/* Desktop styles */
+@media screen and (min-width: 768px) {
+  .search {
+    justify-content: center;
+  }
+
+  .search-company {
+    font-size: 25px;
+  }
+
+  .companies-h1 {
+    font-size: 40px;
+    margin-bottom: 25px;
+  }
+
+  .search-input {
+    width: 400px;
+    padding: 10px;
+  }
+
+  .search-icon {
+    font-size: 37px;
+  }
+}
```

Comment:
- [ ] As part of the requirements, you need to follow the design guidelines, and the only thing that you can customize is the main color, kindly update your app design to match the template

**_your design:_**
![image](https://user-images.githubusercontent.com/89282221/218066497-39f27929-7746-4853-be67-994cf8462af1.png)


_**Teamplate:**_

![image](https://user-images.githubusercontent.com/89282221/218066900-d7a6bad2-efd2-4386-8ccd-b0238e479204.png)

Reply:
Thank you, the icons has been added.